bootstrap/sponge/src/main/java/org/geysermc/platform/sponge/GeyserSpongeConfiguration.java

```diff
@@ -149,6 +149,11 @@ public boolean isCacheChunks() {
         return node.getNode("cache-chunks").getBoolean(false);
     }
 
+    @Override
+    public int getCacheImages() {
+        return node.getNode("cache-skins").getInt(0);
```

Comment:
This should be consistent with the `GeyserJacksonConfiguration` default property.